### ISSUE TITLE
Error  cert-wizzard

### ISSUE THREAD

**Issue by tXambe** [s0lst1c3/eaphammer#190]
Hello,

When execute "./eaphammer --cert-wizard " I obtain this error:

```
/usr/bin/env: �python3.8�: No existe el fichero o el directorio
```

Anyone can help me.

A greeting and thanks

**Comment by tXambe** [comment]
edit eaphammer and change the line #!/usr/bin/env of 3.8 at 3.9

**Comment by minanagehsalalma** [comment]
@tXambe 

![image](https://user-images.githubusercontent.com/20546638/148125983-30556781-1d41-4d37-a0aa-0bfe7036afdc.png)

thanks a lot

**Comment by Pipkill-prog** [comment]
![image](https://user-images.githubusercontent.com/60540072/151885920-67189026-4988-4b27-ba34-7b0a4df88cd3.png)

On Raspbian Lite the packet libxslt.so.1 is missing, so install if needed.
https://etanoox.me/raspberry-pi/importerror-libxslt-so-1-cannot-open-shared-object-file-no-such-file-or-directory/

**Comment by dell-devil** [comment]
> ![image](https://user-images.githubusercontent.com/60540072/151885920-67189026-4988-4b27-ba34-7b0a4df88cd3.png)
> 
> On Raspbian Lite the packet libxslt.so.1 is missing, so install if needed. https://etanoox.me/raspberry-pi/importerror-libxslt-so-1-cannot-open-shared-object-file-no-such-file-or-directory/

Hey , I am also getting the same issue but the resource you gave didn't help me and other solutions ?
![eaphammer-error](https://github.com/s0lst1c3/eaphammer/assets/65786119/1f4cb5d5-8008-486b-bbe9-34134c9e4c77)

**Comment by s0lst1c3** [comment]
Resolved 1.14.0
